src/bp/lang-server/util.ts

```diff
@@ -1,12 +1,28 @@
-import { BadRequestError, NotReadyError, UnauthorizedError } from 'core/routers/errors'
 import { Request } from 'express'
 import _ from 'lodash'
 
+import { BadRequestError, NotReadyError, UnauthorizedError } from '../core/routers/errors'
+
 import LanguageService from './service'
 
 const debugAuth = DEBUG('api:auth')
 
-export const authMiddleware = (secureToken: string) => (req, _res, next) => {
+export const isAdminToken = (req, adminToken: string) => {
+  if (!adminToken || !adminToken.length) {
+    return true
+  }
+  if (!req.headers.authorization) {
```

Comment:
I'd add `|| req.headers.authorization.indexOf(' ') >= 0` to prevent nasty errors on line 17 when there's no space (deconstructor will throw)